airgun/session.py

```diff
@@ -182,8 +181,8 @@ def __enter__(self):
             self._factory.post_init()
 
             # Navigator
-            self.navigator = copy.deepcopy(navigator)
-            self.navigator.browser = self.browser
+            self.navigator = Navigate(self.browser)
```

Comment:
Please note: that we need to create a new Navigator with the session browser for tests that may need many UI sessions and each navigator browser have to bind to the relevant session browser:
example in robottelo many tests use this scenario:
```python
with Session(username=admin_user) as admin_session:
    ...
    with Session(username=non_admin_user) as user_session:
         ...
```